prefect/run_omop_es.py

```diff
@@ -50,12 +50,12 @@ def use_prod_if(condition: bool):
         yield "docker-compose.prod.yml"
 
 
-@flow(flow_run_name=name_with_timestamp, log_prints=True)
+# use string concatenation to avoid processing of curly brackets
+@flow(flow_run_name="{" + "settings_id" + "}-" + get_flow_datetime(), log_prints=True)
```

Comment:
```suggestion
@flow(flow_run_name="{settings_id}-" + get_flow_datetime(), log_prints=True)
```

This should work fine, no need to concatenate the `{}` 🙂